docs/walkthrough/working-with-the-catalog.md

```diff
@@ -39,39 +48,52 @@ Files are the content of your package.
 
 The associated workflow contains the rules for validating your package.
 
-The metadata can be added with JSON editor, represented as a key/value table with infinite nesting. If workflow contains JSON schema, you will have predefined key/value pairs according to the schema.
+The metadata can be added with JSON editor, represented as a key/value table
+with infinite nesting. If workflow contains JSON schema, you will have
+predefined key/value pairs according to the schema.
 
 #### JSON editor
 
-To add a new key/value field double click on an empty cell and type key name, then press "Enter" or "Tab", or click outside of the cell. To change value double click on that value.
+To add a new key/value field double click on an empty cell and type key name,
+then press "Enter" or "Tab", or click outside of the cell. To change value
+double click on that value.
 
-Values can be strings, numbers, arrays, or objects. Every value that you type will be parsed as JSON.
+Values can be strings, numbers, arrays, or objects. Every value that you type
+will be parsed as JSON.
 
 We don't support references and compound types yet.
 
 ### Push to bucket
 
-You can push the existing package from one bucket to another. To use this feature consult [workflows](../advanced-features/workflows.md) page.
+You can push the existing package from one bucket to another. To use this
+feature consult [workflows](../advanced-features/workflows.md) page.
 
 ### Summarize
 
-Adding a `quilt_summarize.json` file to a data package (or S3 directory path) will enable content preview right on the landing page.
+Adding a `quilt_summarize.json` file to a data package (or S3 directory path)
+will enable content preview right on the landing page.
```

Comment:
Add a link to catalog/visualizationdashboards#quilt_summarize.json which is a more comprehensive overview.